CMakeLists.txt

```
@@ -144,3 +144,5 @@ install(
     DESTINATION
     ${LCHIP_INSTALL_CONFIG_DIR}
     )
+
+include(tests/CMakeLists.txt)
```

```suggestion
add_subdirectory(tests)
```

`add_subdirectory` is the more CMake-way to add a target. It does have some caveats though:

* You'll need to make the source paths in `tests/CMakeLists.txt` relative to the actual folder structure.
* You'll need to add this to the end of `tests/CMakeLists.txt`:
  ```cmake
  set_target_properties(
        unit-test
        PROPERTIES
        RUNTIME_OUTPUT_DIRECTORY
        "${PROJECT_BINARY_DIR}"
  )
  ```
  * Although, if you follow the Catch2 v2->v3 migration guide, we can switch to running tests using `ctest` from inside the build directory.